src/api/firebase.js

```diff
@@ -87,10 +87,56 @@ export async function updateItem(listId, itemId, checked) {
 	const listItemRef = doc(db, listId, itemId);
 	const listItemSnap = await getDoc(listItemRef);
 
+	// Retrieving values stored in firebase
 	let totalPurchases = listItemSnap.data().totalPurchases;
+	let dateLastPurchased = listItemSnap.data().dateLastPurchased;
+	let dateCreated = listItemSnap.data().dateCreated;
+	let dateNextPurchased = listItemSnap.data().dateNextPurchased;
 
+	// Function estimating previous purchase interval
+	// Because our lists have updated dateLastPurchased without updated dateNextPurchased,
+	// the time intervals won't be accurate, so we recommend using only:
+	const temporaryPreviousEstimate = getDaysBetweenDates(
```

Comment:
Thank you for being honest! Sarah may have a better explanation for the logic behind this, but I believe we were afraid that because we've been manipulating dates in the back-end in Firebase that the calculations may be off. We ended up removing the temporary anyways so hopefully it's more readable.

Reply:
Basically our thinking was that because dateLastPurchased may be updated in lists we've used for the last few weeks but dateNextPurchased has not updated, the previous time interval estimate wouldn't be correct! But like Alex said, it made more sense to take it out :)